blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+  variable updates on other Dasharo platform. The advantage of the update is a
+  cleaned up code with better quality.
+
+- [Improved visual comfort in the boot manager](https://github.com/Dasharo/dasharo-issues/issues/286)
+
+  Minor visual improvement reported by one of laptop users unable to
+  distinguish the entries in the Boot Manager boot device selection window. Now
+  the currently selected device is indicated by an arrow and highlighted with
+  blue color.
+
+### Fixed
+
+- [MSI PRO Z690-A WIFI DDR4 doesn't initialize IGP on certain Processors models](https://github.com/Dasharo/dasharo-issues/issues/274)
+
+  This fix is a crucial one in this release. Many users reported non-working
+  on-board displays with different CPUs than those which were tested in 3mdeb
+  lab. Now all Alder Lake-S graphics devices should be initialized properly and
+  give firmware output on the screen.
+
+- [MSI PRO Z690-A WIFI DDR4 with two Video Cards (2x Radeon 5600XT) has issues related to MMIO resource allocation](https://github.com/Dasharo/dasharo-issues/issues/245)
+
+  Quite specific issue with board configuration overloaded with GPU cards in
+  the community. As such configuration is not common, the debugging involved
+  the issue reporter cooperation by flashing custom binaries and providing
+  debug logs from Dasharo firmware. It happened that the memory requested by
+  both of the cards was too big to fit into the 32 bit space. The fix for this
+  one required reserving much more space for devices' memory in the 32 bit
+  space, by shifting some of the usable RAM in 32 bit space to 64 bit space.
+  Such change let the firmware allocate all memory resources required by both
+  GPUs.
+
+- [Suspend doesn't work in Qubes OS with v1.1.0](https://github.com/Dasharo/dasharo-issues/issues/271)
+
+  Another issue discovered by community. One of Dasharo Team changes introduced
+  an regression in the suspend/resume feature. All OSes were affected. Fix has
+  been applied and the suspend/resume should now work on all OSes. The tests
+  have been also extended with multiple suspend/resume cycles to avoid such
+  problem in the future.
+
+- [Intel XTU on Windows reports "The platform does not support overclocking" on the MSI PRO Z690-A WIFI DDR4 with a K-series Processor](https://github.com/Dasharo/dasharo-issues/issues/159)
+
+  The issue has been reported by community in the first days after v1.0.0
+  release. Surprisingly the Intel XTU utility reported that platform does not
+  support overclocking, despite all components had all the overclocking
+  capabilities. The culprit was the OC lock bit being set by default in FSP
+  which prevented Intel XTU from modifying any settings from Windows level, so
+  it reported the platform as unsupported. The OC lock changes the
+  initialization path inside FSP, so you may encounter some unexpected changes
+  in CPU configuration. Please do not hesitate to report them. One of such
```

Comment:
how to report?

Reply:
https://github.com/3mdeb/news-and-ideas/pull/513/commits/0f29e7a980e1814bfdf6c3d8ab045c7ba6d63782#diff-b67e40bb2c6780be37b7e6cf6fc2a2a9f9ad977388795cc0c5bdfa6576dc52b6R133